src/components/SelectionList/Search/ActionCell.tsx

```diff
@@ -15,15 +15,17 @@ type ActionCellProps = {
     action?: string;
     isLargeScreenWidth?: boolean;
     isSelected?: boolean;
+    isChildListItem?: boolean;
+    parentAction?: string;
 };
 
-function ActionCell({onButtonPress, action = CONST.SEARCH.ACTION_TYPES.VIEW, isLargeScreenWidth = true, isSelected = false}: ActionCellProps) {
+function ActionCell({onButtonPress, action = CONST.SEARCH.ACTION_TYPES.VIEW, isLargeScreenWidth = true, isSelected = false, isChildListItem = false, parentAction = ''}: ActionCellProps) {
```

Comment:
Do we need `parentAction`? I think we always wanna skip this logic if `isChildListItem=true`, no?